src/picongpu/include/plugins/radiation/radFormFactor.hpp

```diff
@@ -44,17 +44,18 @@ namespace picongpu
        */
       HDINLINE float_X operator()(const float_X N, const float_X omega, const vector_X observer_unit_vec) const
       {
+        //util::Pow<float_X, 1> pow;
+        const float_X sincX = util::pow( math::sinc( observer_unit_vec.x() * CELL_WIDTH / ( SPEED_OF_LIGHT * float_X( 2.0 ) ) * omega ), 1 );
```

Comment:
If the shapes only differ in the used pow function than please show it in your code:
```C++
template< uint32_t T_order >
struct BaseShape
{
   HDINLINE float_X operator()(const float_X N, const float_X omega, const vector_X observer_unit_vec) const
  {
     //....
     // pow(T_order);
  }
};

struct radFormFactor_TSC_3D : public BaseShape< 2 >
{

};
```

Maybe a function parameter with the power factor is also possible.